Dockerfile

```diff
@@ -17,4 +20,5 @@ COPY . .
 
 RUN chmod +x ./bin/hash-honeypot
 
-ENTRYPOINT [ "./bin/hash-honeypot" ]
\ No newline at end of file
+#ENTRYPOINT [ "./bin/hash-honeypot", "help" ]
+CMD ["sleep", "infinity"]
```

Comment:
Could be wrong with this, but doesn't this mean nothing will be served?

`CMD` will overwrite parent container's (`node:18`) `CMD` and `ENTRYPOINT` thus I don't think anything is actually starting the node app. Most examples I am seeing online use something like `CMD ["node", "index.js"]` as `CMD`

Reply:
You are correct, but this is intended because Hash is executed as CLI nodejs app not a long running one. So we needed to keep the image a live with `sleep` and then when we want to execute the cli we run it like this
```
docker-compose exec hash-honeypot /usr/src/app/bin/hash-honeypot help
```

Reply:
In another words, this container is just for local testing. So we needed the flexibility to run hash CLI when we need to.